static/js/controllers/configuration.js

```diff
@@ -41,6 +41,14 @@ angular.module('inboxControllers').controller('ConfigurationCtrl',
         }
       },
       {
+        state: 'configuration.upgrade',
+        icon: 'fa-rocket',
```

Comment:
FTR we discussed this and I don't think we came up with a better one.

I picked the rocket because to me it implies a big technical thing, danger (launching rockets doesn't always go to play), progress, something that might take a long time (we're going to spaaaaaaaaaaaaaaaaaaaaaace), traveling to new things,  etc.